src/util.ts

```diff
@@ -12,4 +14,18 @@ const aqiCutoffs = {
   veryUnhealthy: 300, // Very unhealthy for all individuals (201-300)
 };
 
+/**
+ * Interface for a Selected Sensor. This is used in the Home page to keep track
+ * of the values for the sensor that is currently selected on the map.
+ */
+interface SelectedSensor {
+  purpleAirId: number;
+  sensorDocId: string;
+  name: string;
+  aqi: string;
+  isAqiValid: boolean;
```

Comment:
nit: use `isValid` rather than `isAqiValid` just for consistency throughout the codebase. We could add tsdoc with a list of the values that further clarifies what `isValid` means